app/controllers/miq_policy_controller/policies.rb

```diff
@@ -225,7 +225,7 @@ def excluded_event?(event)
 
   def policy_get_all_folders(parent = nil)
     if !parent.nil?
-      @folders = %w(Host Vm)
+      @folders = MiqPolicy::UI_FOLDERS.collect(&:name)
```

Comment:
See above, this is fine but feels like this should be doing i18n calls on the class names.